tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -193,7 +190,96 @@ def test_positive_hypervisor_id_option(
                 command, form_data['hypervisor_type'], org=default_org.label
             )
             assert get_configure_option('hypervisor_id', config_file) == value
-        virtwho_config.delete()
-        assert not target_sat.api.VirtWhoConfig().search(
-            query={'search': f"name={form_data['name']}"}
+
+    @pytest.mark.tier2
+    @pytest.mark.parametrize('deploy_type', ['id', 'script'])
+    def test_positive_prism_central_deploy_configure_by_id_script(
+        self, default_org, form_data, target_sat, deploy_type
+    ):
+        """Verify "POST /foreman_virt_who_configure/api/v2/configs" on nutanix prism central mode
+
+        :id: e7652f64-eaf8-45a5-ac01-eb40d53b6603
+
+        :expectedresults:
+            Config can be created and deployed
+            The prism_central has been set in /etc/virt-who.d/vir-who.conf file
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+        """
+        form_data['prism_flavor'] = "central"
+        virtwho_config = target_sat.api.VirtWhoConfig(**form_data).create()
```

Comment:
Why not using `virtwho_config` fixture here instead?

Reply:
> Why not using `virtwho_config` fixture here instead?

@Gauravtalreja1 Because virtwho_config fixture will create prsim-flavor == element config, this case need to create prisim-flavor == central config. the same as in https://github.com/SatelliteQE/robottelo/pull/10016#discussion_r980623550